Annex60/Fluid/Sensors/FlowVelocity.mo

```diff
@@ -0,0 +1,122 @@
+within Annex60.Fluid.Sensors;
+model FlowVelocity "Ideal sensor for flow velocity"
```

Comment:
I suggest to rename it the model to `Velocity` which is shorter and in my view, it is clear that it is the velocity of the flow as the sensor is in the `Fluid` package.